Microsoft.Research/RegressionTest/ClousotTests/ClousotTests.csproj

```diff
@@ -179,6 +179,12 @@
     <None Include="..\TestFrameworkOOB\UserFeedback.cs">
       <Link>Sources\UserFeedback.cs</Link>
     </None>
+    <Compile Include="..\IteratorAnalysis\IteratorSimpleContract\IteratorSimpleContract.cs">
+      <Link>Sources\IteratorSimpleContract.cs</Link>
+    </Compile>
+    <Compile Include="..\TestPostconditionInference\Filtering.cs">
+      <Link>Sources\Filtering.cs</Link>
```

Comment:
@sharwell I prefer separate PR, but separate commits are fine for now.

@tom-englert What do you think about @panacekcz comment? We can fix this in separate commit/PR.